assets/src/dashboard/app/views/shared/storyListView.js

```diff
@@ -363,7 +363,7 @@ export default function StoryListView({
                         tabIndex={0}
                         onFocus={onFocusSelectAll}
                         onBlur={onBlurDeselectAll}
-                        size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.XXX_SMALL}
+                        size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.XX_SMALL}
```

Comment:
this was intentionally updated to XXX_SMALL, it's supposed to be 14px:
<img width="789" alt="Screen Shot 2021-03-16 at 9 06 27 AM" src="https://user-images.githubusercontent.com/10720454/111341626-ea2b6300-8636-11eb-98c8-a6f07e2e6e00.png">
![Screen Shot 2021-03-16 at 9 07 14 AM](https://user-images.githubusercontent.com/10720454/111341732-04fdd780-8637-11eb-9b2f-7a5f0fc313e4.png)